send.ts

```diff
@@ -59,7 +59,7 @@ function isHidden(root: string, path: string) {
 
 async function exists(path: string): Promise<boolean> {
   try {
-    return (await Deno.stat(path)).isFile();
+    return Deno.stat(path).then(x => x.isFile);
```

Comment:
the try/catch block is not used without await. Do not use .then for this case.

Reply:
Taking note
...
Done